tslint.json

```diff
@@ -13,6 +13,7 @@
     "no-namespace": false,
     "quotemark": false,
     "unified-signatures": false,
-    "variable-name": false
+    "variable-name": false,
+    "no-shadowed-variable": false
```

Comment:
I actually strongly agree with this rule (and the rule above), shadowing variables is pretty bad, can we just use another variable name instead of disabling the rule 😄

Reply:
It doesn't make much sense in a declarations file.
```ts
namespace Electron {
  declare const webContents: ...;
  class BrowserWindow {
    static fromWebContents(webContents: WebContents): BrowserWindow;
  }
}
```
The error in on `webContents`, which seems like a non-issue, and as it's generated from the .json, changing it is non-trivial.

Reply:
@NaridaL Lel, thought this was linting our code / tests not the output 👍